pwiz_tools/Skyline/FileUI/ManageResultsDlg.cs

```diff
@@ -144,22 +144,22 @@ protected override AuditLogEntry CreateEntry(SrmDocumentPair docPair)
 
                 if (_removedAllLibraryRuns)
                 {
-                    entry = entry.AppendAllInfo(new[] { new MessageInfo(MessageType.removed_all_libraries) });
+                    entry = entry.AppendAllInfo(new[] { new MessageInfo(MessageType.removed_all_libraries, docPair.OldDoc.DocumentType) });

```

Comment:
I generally use old since that describes the state when the change was initiated. In some cases, though, the change is that the old document was empty so it's not useful for knowing doc type.